tests/test_ase_traj.py

```diff
@@ -48,6 +48,8 @@ class TestASEtraj3(unittest.TestCase, CompLabeledSys, IsPBC):
     def setUp(self):
         self.system_1 = dpdata.System("ase_traj/Cu.traj", fmt="ase/traj")
         self.system_2 = dpdata.LabeledSystem("ase_traj/Cu.traj", fmt="ase/traj")
```

Comment:
the two systems are used to compare the outcomes of your code. 
So the first one is parsing from the `ase_traj`, 
```
self.system_1 = dpdata.LabeledSystem("ase_traj/Cu.traj", fmt="ase/traj")
```
the second one is the numerical outcomes stored as deepmd format. 
self.system_1 = dpdata.LabeledSystem("results you expect to be", fmt="deepmd/npy")